pages/cloud/dedicated/partitioning_ovh/guide.en-gb.md

```diff
@@ -0,0 +1,294 @@
+
+|Error message|Details|Solution(s)|
+|---|---|---|
+|Some Linux distributions such as RHEL family OSes don't support those mountpoints / mountpoint reserved/managed by OVHcloud (\*\*\*). Please remove those mountpoints and restart an installation|- You have chosen `/boot/efi` as mountpoint. OVHcloud will create this partition automatically for you if your server needs one<br />- You have chosen a mountpoint that is symlinked on some operating systems. See [Filesystem Hierarchy Standard](https://refspecs.linuxfoundation.org/fhs.shtml) for more details.|- Choose another mountpoint for the partition or remove this partition from your partitioning layout|
+|Partition of type \*\*\* with mountpoint \*\*\* cannot fill the disk.|- You have chosen the `swap` partition to fill the disk, we disallow this to avoid creating unnecessarily large `swap` partitions|- Set a fixed size for the `swap` partition|
+|Missing `/` partition. Please add a `/` partition in your partition scheme!|- Any Linux-based OS needs at least a `/` partition|- Add a `/` partition in your partitioning layout|
+|\*\*\*. Please adjust partitions so that the \*\*\* partition fits on \*\*\* disk(s)|- You have chosen a partition with a RAID that requires a number of disks that your server can provide, but some disks are already full because of other partitions and/or this current partition|- If it is not already set on another partition, set the partition size to fill the disk<br />- Reduce the size of this partition so that it fits the disk(s)<br />- Reduce the size of other partitions partitions so that this partition fits the disk(s)|
+
+#### Input customer auto-fixing
+
+In order to improve customer experience, reduce [OVHcloud support](https://help.ovhcloud.com/en-gb/) workload and to avoid introducing breaking changes for customer, some customer input is automatically fixed or changed by the backend. The following table gives an overview of what is currently auto-fixed/changed during the **pre-processing**:
+
+|Subject|Description|
+|---|---|
+|ZP grouping|Any ZFS filesystem partitions with the same RAID level will be grouped within the same zpool (ZP) (if possible depending of the size of the disk(s))|
+|LV grouping|All partitions of type `lv` with the same RAID level will be grouped within the same VG (if possible depending of the size of the disk(s))|
+|VG expanding|In case of LV partitions with a RAID level of 0, the VG will span multiple PPs (therefore PDs) and no SR device will be created|
+|VG Disk fill|Remaining disk space will be filled by a VG (if any LV exists). The size of LVs attached to the VG is not affected.|
+|RAID level reducing|In case the customer chose a partition with a RAID level that requires more disks than the server has, the RAID level will be automatically reduced in the following order: 6, 10, 5, 1, 0 (or 7 (raidz3), 6 (raidz2), 5 (raidz), 1 (mirror), 0 (striped VDev) for ZFS)|
```

Comment:
```suggestion
|RAID level reducing|In case the customer chose a partition with a RAID level that requires more disks than the server has, the RAID level will be automatically reduced in the following order: 6, 10, 5, 1, 0 (or 7 (raidz3), 6 (raidz2), 5 (raidz), 1 (mirror), 0 (striped vdev) for ZFS)|
```